tests/behat/features/plugin-check.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Test that the WP-CLI command works.
+
+  Scenario: Check a non-existent plugin
```

Comment:
I really wish we had a test of a plugin that works. Just testing failure here feels weird.

Reply:
Right now this PR just lays the foundation for the tests. New tickets can be created to add a few more tests.

Reply:
How can we ensure the foundation work if the tests do not do anything meaningful?

Reply:
There was lots of manual testing of the CLI command so far, and the tests setup is standard for WP-CLI and battle-tested. Are you looking for more?